test/testClient.js

```diff
@@ -8,4 +8,8 @@ describe('API client', function () {
     it('can be constructed', function () {
         void new Cog.Client();
     });
+
+    it('can be constructed with params', function () {
+        let client = new Cog.Client({'region': 'us-east-2'});
```

Comment:
Style nitpicks: no unused variables (use `void new Cog.Client(...)` as above) and use unquoted property names when possible.